### PR TITLE
Add FAQ entry to the template checklists

### PR DESCRIPTION
**Pull request by opusforlife2**
#### What is it?
- [ ] Bugfix (user facing)
- [ ] Feature (user facing)
- [ ] Codebase improvement (dev facing)
- [x] Meta improvement to the project (dev facing)

#### Description of the changes in your PR

- Added reading the FAQ to the checklist.

#### Fixes the following issue(s)

- Hopefully leads to fewer duplicate issues.

#### Due diligence
- [x] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/question.yml [hunk 1]
```diff
@@ -16,6 +16,8 @@ body:
       options:
         - label: "I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to."
           required: true
+        - label: "I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed."
```
Comment (mhmdanas) [inline .github/ISSUE_TEMPLATE/question.yml:19]:
```suggestion
        - label: "I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed."
```

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 2]
```diff
@@ -16,6 +16,8 @@ body:
       options:
         - label: "I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to."
           required: true
+        - label: "I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed."
```
Comment (mhmdanas) [inline .github/ISSUE_TEMPLATE/feature_request.yml:19]: "problem" doesn't quite fit here in the context of a feature request, any suggestions?

**Review by mhmdanas — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.yml (hunk 2 above)
Comment (opusforlife2) [reply .github/ISSUE_TEMPLATE/feature_request.yml:19]: They're trying to solve a problem, though. That's why we have "Describe any problem or limitation you come across while using the app which would be solved by this feature." below.

Comment (mhmdanas) [reply .github/ISSUE_TEMPLATE/feature_request.yml:19]: Hmm, does that apply to my other review comment too?

Comment (mhmdanas) [reply .github/ISSUE_TEMPLATE/feature_request.yml:19]: Okay then, go ahead and commit that suggestion.

**Review by mhmdanas — APPROVED** [review]

**Pull request merged**